rllib/algorithms/ppo/ppo.py

```diff
@@ -439,8 +439,28 @@ def training_step(self) -> ResultDict:
                         worker_set=self.workers,
                         max_env_steps=self.config.train_batch_size,
                     )
+                # TEST
+                #import pickle
+                #import torch
+                #import os
+                #with open("train_batch.pkl", "wb") as file:
+                #    pickle.dump(train_batch, file)
+                #torch.save(self.workers.local_worker().policy_map["default_policy"].model.state_dict(), "model_weights.pth")
+                # END TEST
+
```

Comment:
remove?

Reply:
Sorry, will comb through changes and clean everything up before merging. Thanks for this catch!

Reply:
done